telegram/_chat.py

```diff
@@ -355,6 +359,92 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["Chat"]:
 
         return super()._de_json(data=data, bot=bot, api_kwargs=api_kwargs)
 
+    def mention_markdown(self, name: str = None) -> str:
+        """
+        .. versionadded:: 20.0
+
+        Note:
+            :tg-const:`telegram.constants.ParseMode.MARKDOWN` is a legacy mode, retained by
+            Telegram for backward compatibility. You should use :meth:`mention_markdown_v2`
+            instead.
+
+        Args:
+            name (:obj:`str`): The name used as a link for the chat. Defaults to :attr:`full_name`.
+
+        Returns:
+            :obj:`str`: The inline mention for the chat as markdown (version 1).
+
+        """
+        full_name = self.full_name
```

Comment:
do you actually need this shortcut? full_name is an attribute, so no performance gain, and self. doesn't make much of a difference in code length imo.